src/dsp_tools/commands/start_stack.py

```diff
@@ -90,12 +89,16 @@ def _get_url_prefix(self) -> str:
         """
         url_prefix_base = "https://github.com/dasch-swiss/dsp-api/raw"
 
+        docker_compose_pth = importlib.resources.files("dsp_tools").joinpath("resources/start-stack/docker-compose.yml")
+        docker_compose = yaml.safe_load(docker_compose_pth.read_bytes())
+        tag = docker_compose["services"]["api"]["image"].split(":")[-1]
+
+        response = requests.get(f"https://api.github.com/repos/dasch-swiss/dsp-api/git/ref/tags/{tag}", timeout=5)
+        commit_of_used_api_version = response.json()["object"]["sha"]
+
         if self.__stack_configuration.latest_dev_version:
             return f"{url_prefix_base}/main/"
```

Comment:
Nitpick: It would probably make sense to move this guard up to the beginning of the function. It does not depend on the result of the above lines which include file system and HTTP calls, so if the condition is true, it would make sense not to have to do this work.

Reply:
Absolutely